OpenDreamClient/Interface/Controls/ControlWindow.cs

```diff
@@ -336,7 +339,11 @@ public override void SetProperty(string property, string value, bool manualWinse
         switch (property) {
             case "size":
                 if (_myWindow.osWindow is {ClydeWindow: not null}) {
-                    _myWindow.osWindow.ClydeWindow.Size = new DMFPropertySize(value).Vector;
+                    var size = new DMFPropertySize(value);
+                    var uiScale = _configurationManager.GetCVar(CVars.DisplayUIScale);
```

Comment:
A few different things can affect the UI scale. This CVar is actually 0 by default, to indicate using your OS settings.
```suggestion
                    var uiScale = _myWindow.osWindow.UIScale;
```